extension/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "vectorize"
-version = "0.20.0"
+version = "0.20.1"
```

```suggestion
version = "0.21.0"
```

Since our intention here is to add a new feature while maintaining backwards compatibility. https://semver.org/#summary